region/crateria/west.json

```diff
@@ -1389,6 +1389,38 @@
                 }
               ]
             }
+          ],
+          "leaveWithGModeSetup": [
+            {
+              "strats": [
+                {
+                  "name": "Get hit by Waver",
+                  "notable": false,
+                  "requires": [
+                    {"or": [
+                      "h_canDestroyBombWalls",
+                      {"canComeInCharged": {
+                        "fromNode": 1,
+                        "framesRemaining": 180,
+                        "shinesparkFrames": 0
+                      }},
+                      {"and": [
+                        {"comeInWithGMode": {
+                          "fromNodes": [1],
+                          "mode": "any",
+                          "artificialMorph": true
+                        }},
+                        {"or": [
+                          "Bombs",
+                          {"ammo": { "type": "PowerBomb", "count": 1 }}
+                        ]}
+                      ]}
+                    ]}
+                  ],
+                  "devNote": "FIXME: It is not currently possible to represent coming in from the opposite door blue or in an artificial morph."
```

Comment:
Artifical morph can't overload PLMs to cross the room. Bombing blocks would make them solid.